### ISSUE TITLE
Why scaling lr with batch_size*world_size/256?

### ISSUE THREAD

**Issue by SunDoge** [NVIDIA/apex#407]
[https://github.com/NVIDIA/apex/blob/master/examples/imagenet/main_amp.py#L143](https://github.com/NVIDIA/apex/blob/master/examples/imagenet/main_amp.py#L143)

Does it mean that `args.lr` is the learning rate based on batch size 256? 
Can I use it to scale the lr of `Adam`?

**Comment by seongwook-ham** [comment]
i think it's due to large batch training heuristic. see 

> https://research.fb.com/wp-content/uploads/2017/06/imagenet1kin1h5.pdf

for good generalization, we have to increase learning rate proportional to batch size
